cmds/minigames/connect4.js

```diff
@@ -175,288 +289,320 @@ class connect4 extends two_p_Game{
+        let color = board[y][x];
         if (color === "⚪") return 1;
         //horizontal
-        if(this.search(x, y , color, 1, 0, 0) + this.search(x, y , color, -1, 0, 0) - 1 >= 4){
-            return 2
+        if (
+            this.search(x, y, color, 1, 0, 0) +
+            this.search(x, y, color, -1, 0, 0) - 1 >= 4
+        ) {
+            return 2;
         }
         //lt-rb diagonal
-        if(this.search(x, y , color, 1, 1, 0) + this.search(x, y , color, -1, -1, 0) - 1 >= 4){
-            return 2
+        if (
+            this.search(x, y, color, 1, 1, 0) +
+            this.search(x, y, color, -1, -1, 0) - 1 >= 4
+        ) {
+            return 2;
         }
         //lt-rb diagonal
-        if(this.search(x, y , color, -1, 1, 0) + this.search(x, y , color, 1, -1, 0) - 1 >= 4){
-            return 2
+        if (
+            this.search(x, y, color, -1, 1, 0) +
+            this.search(x, y, color, 1, -1, 0) - 1 >= 4
+        ) {
+            return 2;
         }
         //vertical
-        if(this.search(x, y , color, 0, 1, 0) + this.search(x, y , color, 0, -1, 0) - 1 >= 4){
-            return 2
+        if (
+            this.search(x, y, color, 0, 1, 0) +
+            this.search(x, y, color, 0, -1, 0) - 1 >= 4
+        ) {
+            return 2;
         }
 
-        let countstack = 0
-        for(let i = 0; i < 7; i++){
-            if(this.stack[i] === -1){
-                countstack++
+        // check how many pieces in a column
+        let countstack = 0;
+        for (let i = 0; i < 7; i++) {
+            if (this.stack[i] === -1) {
+                countstack++;
             }
         }
 
-        if(countstack === 7) return 3;
-
-        return 1
+        // a column is full
+        if (countstack === 7) return 3;
 
+        return 1;
     }
 
-    
-
-    search(x, y, color, dx, dy, count, board = this.map){
-
-        if(x<0 || x>6 || y<0 || y>5) return count
-        if(color !== board[y][x]) return count
-
-        return this.search(x+dx, y+dy, color, dx, dy, count+1, board)
+    // (not sure what this whole code does)
+    search(x, y, color, dx, dy, count, board = this.map) {
+        if (x < 0 || x > 6 || y < 0 || y > 5) return count;
+        if (color !== board[y][x]) return count;
 
+        return this.search(x + dx, y + dy, color, dx, dy, count + 1, board);
     }
 
-    search2(x, y, color, dx, dy, count, board){
-
-        if(x<0 || x>6 || y<0 || y>5) return {count : count, live : 0}
-        if(board[y][x] !== color){
-            if(board[y][x] === "⚪"){
-                return {count : count, live : 1}
+    search2(x, y, color, dx, dy, count, board) {
+        if (x < 0 || x > 6 || y < 0 || y > 5) return { count: count, live: 0 };
+        if (board[y][x] !== color) {
+            if (board[y][x] === "⚪") {
+                return { count: count, live: 1 };
             }
-            return {count : count, live : 0}
+            return { count: count, live: 0 };
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/xiaomiao-fong/Chen_bot/pullRequest?prid=7807535)
```suggestion
            return { count, live: 0 };
```